### PR TITLE
PXB-762 LP #1621184: the recovery process with relay-log-info-repository=TABLE should be documented

### PR DESCRIPTION
**Pull request by patrickbirch**
…ory=TABLE should be documented (2.4)

Added a note to the full backup restore section about enabling relay-log-recovery

### CODE REVIEW

**Review by kathy-williamson — APPROVED** [review]

File: storage/innobase/xtrabackup/doc/source/backup_scenarios/full_backup.rst [hunk 1]
```diff
@@ -185,6 +185,18 @@ option, which will copy the backup to the server's :term:`datadir`:
 If you don't want to save your backup, you can use the :option:`xtrabackup
 --move-back` option which will move the backed up data to the :term:`datadir`.
 
+.. note::
+
+    When ``relay-log-info-repository=TABLE`` is enabled, the instance recovered from the backup has errors in the error log, like the following:
```
Comment (rahulmalik87) [inline storage/innobase/xtrabackup/doc/source/backup_scenarios/full_backup.rst]: does it require for 8.0 or only for 5.7?

Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/backup_scenarios/full_backup.rst]: relay-log-info-repository is a variable in 8.0 and 5.7
https://dev.mysql.com/doc/refman/8.0/en/replication-options-replica.html#sysvar_relay_log_info_repository

Comment (altmannmarcelo) [reply storage/innobase/xtrabackup/doc/source/backup_scenarios/full_backup.rst]: @patrickbirch change LGTM, just place it in the end of the doc - https://github.com/percona/percona-xtrabackup/blame/2.4/storage/innobase/xtrabackup/doc/source/backup_scenarios/full_backup.rst#L216

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/backup_scenarios/full_backup.rst [hunk 2]
```diff
@@ -185,6 +185,18 @@ option, which will copy the backup to the server's :term:`datadir`:
...
+    
+    .. code-block:: bash
+    
+        2019-08-09 12:40:02 69297 [ERROR] Failed to open the relay log '/data/mysql-relay-bin.004349' (relay_log_pos 5534092)
+    
+    To avoid these types of issues, enable ``relay_log_recovery``.
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/backup_scenarios/full_backup.rst]: Also add:

Or execute `RESET SLAVE` prior to `CHANGE MASTER TO`

Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/backup_scenarios/full_backup.rst]: Added and moved.

**Review by altmannmarcelo — APPROVED** [review]
LGTM

**Pull request merged**